stack.yaml

```diff
@@ -9,7 +9,7 @@ extra-deps:
   - postgresql-error-codes-1.0.1
   - tomland-1.3.3.1
   - validation-selective-0.1.0.1
-  - git: git@github.com:haskell-servant/servant.git
+  - git: https://github.com/haskell-servant/servant.git
```

Comment:
why is it better to use the `https` version instead of the `ssh` one?

Reply:
It's not necessarily better, I switched laptops and didn't realize I had to setup ssh keys again. This was just a quick fix that I forgot to undo. But it's undone now :)

Reply:
:+1: